torch/csrc/distributed/rpc/rpc_agent.h

```diff
@@ -38,6 +40,10 @@ struct WorkerId {
         name_);
   }
 
+ bool operator==(const WorkerId& other) {
+   return name_ == other.name_ && id_ == other.id_;
+ }
+
```

Comment:
If a worker is identified by just the `uint16_t`, shouldn't the name be stored some place else? Or not show up in the equivalence check? With this equivalence, you can have `Worker("foo", 1)` and `Worker("bar", 1)` coexist, which I believe would be malformed. @mrshenli is this intentional or just to make coding it simpler?